app/certificate_expiry_check.py

```diff
@@ -98,10 +63,22 @@ def send_email(email_type, params):
 
 
 def main():
+    # Instantiate services
     s3_service = S3Service()
+    gandi_service = GandiService()
+
+    # Get a list of the email mappings from S3
     email_mappings = s3_service.get_json_file(config['s3']['s3_bucket_name'], config['s3']['s3_object_name'],
                                               './app/s3_email_mapping.json')
-    find_expiring_certificates(email_mappings)
+
+    # Get a list of the expired certificates from Gandi
+    certificate_list = gandi_service.get_expiring_certificates(config)
```

Comment:
Should we be passing in `email_mappings` here as well?

Reply:
@PepperMoJ Will also need to updated the tests, they aren't passing now since the interface has changed, but also the tests aren't being run in CI/CD due to how the CI/CD runs the tests 🧪